### ISSUE TITLE
KC storage - Unicorn tears

### ISSUE THREAD

**Issue by noxidevad** [iMasonite/KenetiCommunityMod#25]
Noticed late night in server when i was just pottering about the base. 

Upon reporting the alpha unicorn issue from (Issue 23 [https://github.com/iMasonite/KenetiCommunityMod/issues/23](url)), 
I had 194 unicorn tears in the KC dedicated storage and noticed late last night that i had 0 , Willy has also encountered this problem. Unsure about other users

Can you have a look into this as good chance it may not be stopping the timer inside the kc box or something else entirely , workaround obviously for us is just too craft the tears straight into aphrodisiacs for the time being. But i do miss my tears spent ages getting them :(

**Comment by iMasonite** [comment]
yeah its not the timers are the items are stored as a number of items, not an actual item (this is why no item spoils inside dedicated storage) 

There may be something else though, so I'll dig around.

**Comment by iMasonite** [comment]
Unable to replicate an issue with this, the same code is used on all other items so if this is an issue, we should be seeing it with other resources. The Unicorn Tear item is a sub class of Black Pearls, if there was some issue with the item, we would also see it with Black Pearls. 

Black Pearls are a sub class of `Primal Item Resource Generic` that covers just about every resource, we would also see it with those resources.

If its a Mod issue with custom items, this issue would also show up with `Aphrodisiac`, `Creature Mindwipe`, `Super Chill Brew` etc as they all share the same methods to add and store them to dedicated storage.
